### ISSUE TITLE
Missing Stocks

### ISSUE THREAD

**Issue by pranjaljately** [Prubby/covid-19-vs-markets#54]
- Facebook (FB)

**Comment by pranjaljately** [comment]
- Apple (APPL)
- Netflix (NFLX) [stock found but - Cannot read property 'length' of undefined]
- Twitter (TWTR)
- Nvidia (NVDA) [stock found but - Cannot read property 'length' of undefined]
- Zoom (ZM)
- Paypal (PYPL) [stock found but - Cannot read property 'length' of undefined]
- JPMorgan (JPM)
- Boeing Co (BA) [stock found but - Cannot read property 'length' of undefined]
- Adobe (ADBE)
- AT&T (T)
- Expedia (EXPE)
- Fitbit (FIT)
- Qualcomm (QCOM)
- Salesforce (CRM) [stock found but - Cannot read property 'length' of undefined]
- Slack (WORK)
- Starbucks (SBUX)
- Stryker (SYK)
- Uber (UBER)
- UPS (UPS)
- Verizon (VZ)
- Visa (V)
- Walmart (WMT)
- WWE (WWE)
